src/jit/lsraxarch.cpp

```diff
@@ -1978,13 +1978,15 @@ void LinearScan::TreeNodeInfoInitIntrinsic(GenTree* tree)
             }
             break;
 
-#ifdef _TARGET_X86_
+#if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
         case CORINFO_INTRINSIC_Cos:
         case CORINFO_INTRINSIC_Sin:
         case CORINFO_INTRINSIC_Round:
+#if defined(LEGACY_BACKEND)
```

Comment:
This is also not needed, since `NYI_X86` is a no-op for non-X86 targets.

Reply:
It looks to be a no-op for AMD64, but not for `X86`, so I think I still need this particular `#ifdef`: https://github.com/dotnet/coreclr/blob/master/src/jit/error.h#L180